enclave-runtime/Cargo.lock

```diff
@@ -1725,7 +1735,7 @@ dependencies = [
  "itp-storage",
  "itp-types",
  "itp-utils",
- "log",
+ "log 0.4.17",
```

Comment:
you introduced a duplicate dep here.

Reply:
thx. no idea what happened here and I had to edit the lock file manually to fix it. cargo update messes everything up